src/Exceptions/HttpNetworkAuthenticationRequired.php

```diff
@@ -8,4 +8,3 @@ class HttpNetworkAuthenticationRequired extends HttpException {
     public const DESCRIPTION = "Indicates that the client needs to authenticate to gain network access.";
     public const HREF = "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/511";
 }
-
```

Comment:
Removed unix line ending - ALWAYS have a unix line ending, it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces